### PR TITLE
docs: add feature platform use cases

### PR DESCRIPTION
**Pull request by tobegit3hub**
Add docs of openmldb feature platform.

### CODE REVIEW

File: docs/zh/app_ecosystem/feature_platform/quickstart.md [hunk 1]
```diff
@@ -1,40 +1,109 @@
...
+
+![](./images/test_offline_sample_detail.png)
+
+在本地即可查看导出的样本文件内容。为了验证 OpenMLDB 特征平台提供的在线离线一致性，可记录离线特征结果，并于后面的在线特征计算做比较。
+
+![](./images/local_test_offline_samples.png)
+
+### 4. 创建在线服务
+
+在“特征服务”页面可以选择创建特征服务，同样是之需要选择上线的特征，以及提供特征服务名称和版本即可。
```
Comment (Elliezza) [inline docs/zh/app_ecosystem/feature_platform/quickstart.md]: there's a typo, 同样是只需要

File: docs/zh/app_ecosystem/feature_platform/install/config_file.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+## 配置项目
+
+
+| 配置项目 | 介绍 | 类型 | 示例 |
+| ------- | --- | --- | ---- |
+| server.port | 服务端口 | int | 8888 |
+| openmldb.zk_cluster | ZooKeeper 集群地址 | string | 127.0.0.1:2181 |
+| openmldb.zk_path | OpenMLDB 根路径 | string | /openmldb |
+| openmldb.apiserver | OpenMLDB APIServer 地址 | string | 127.0.0.1:90 |
```
Comment (Elliezza) [inline docs/zh/app_ecosystem/feature_platform/install/config_file.md:29]: typo? 127.0.0.1:9080

File: docs/zh/app_ecosystem/feature_platform/functions/offline_scenario.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+
+# 离线场景
+
+## 介绍
+
+用户可以在此界面完成所有跟离线场景有关的需求，功能包括：
+
+* 创建离线样本
+* 查看离线样本
+* 删除服务版本
```
Comment (Elliezza) [inline docs/zh/app_ecosystem/feature_platform/functions/offline_scenario.md:10]: 删除离线样本？

File: docs/zh/app_ecosystem/feature_platform/install/config_file.md (hunk 2 above)
Comment (tobegit3hub) [reply docs/zh/app_ecosystem/feature_platform/install/config_file.md:29]: Should be 127.0.0.1:9080.

File: docs/zh/app_ecosystem/feature_platform/functions/offline_scenario.md (hunk 3 above)
Comment (tobegit3hub) [reply docs/zh/app_ecosystem/feature_platform/functions/offline_scenario.md:10]: yes, it is typo.

**Comment by tobegit3hub** [comment]
Duplicated with https://github.com/4paradigm/OpenMLDB/pull/3705

**Pull request closed without merging**